message-fanout/README.md

```diff
@@ -0,0 +1,23 @@
+# message-fanout
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+`sh
+npm i @winglibs/message-fanout
+`
+
+## Usage
+
+`js
+bring message-fanout;
+
+let adder = new message-fanout.Adder();
+`
```

Comment:
can you add some examples of this adding consumers

Reply:
Now it has a better example, Hasan.